horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/Schema.java

```diff
@@ -74,4 +74,23 @@ public String toString() {
                     '}';
         }
     }
+
+    public static class CreateOrUpdateEvent {
+        public int id;
+        public String uri;
+
+        public CreateOrUpdateEvent() {}
+
+        public CreateOrUpdateEvent(int id, String uri) {
+            this.id = id;
+            this.uri = uri;
+        }
+        @Override
+        public String toString() {
+            return "CreateOrUpdateEvent{" +
+                    "id=" + id +
+                    ", uri=" + uri +
+                    '}';
+        }
+    }
```

Comment:
I don't think we want to expose this class in the API, it is an internal type

Reply:
Yeah, makes sense.. I think I will simplify the event further (it should work with just an Integer as highlighted in the pr description). This way it won't be exposed anywhere as no more existing.

Reply:
Completely removed with commithttps://github.com/Hyperfoil/Horreum/commit/2a5aa2fa27d280b24018c9787b4a8ad27b2737de

Reply:
thanks